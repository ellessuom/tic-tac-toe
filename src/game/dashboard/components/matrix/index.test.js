import React from 'react';
import { shallow } from 'enzyme';
import Matrix from './index';

describe('<Matrix />', () => {
  it('should render Matrix structure correctly', () => {
    const component = shallow(<Matrix />);
    expect(component).toMatchSnapshot();
  });

  it('should contain all 9 tiles', () => {
    const component = shallow(<Matrix />);
    expect(component.children()).toHaveLength(9);
  });
});
