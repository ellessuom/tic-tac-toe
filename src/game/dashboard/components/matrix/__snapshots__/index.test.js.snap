// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Matrix /> should render Matrix structure correctly 1`] = `
<div
  className="matrix-container"
>
  <Tile
    i={0}
    key="0"
  />
  <Tile
    i={1}
    key="1"
  />
  <Tile
    i={2}
    key="2"
  />
  <Tile
    i={3}
    key="3"
  />
  <Tile
    i={4}
    key="4"
  />
  <Tile
    i={5}
    key="5"
  />
  <Tile
    i={6}
    key="6"
  />
  <Tile
    i={7}
    key="7"
  />
  <Tile
    i={8}
    key="8"
  />
</div>
`;
